src/db/util/sequelize-model.ts

```diff
@@ -72,6 +73,16 @@ export type AdditionalFindArgsForSequelizeTables = {
   includeDeleted?: true;
 };
 
+export type AdditionalDestroyArgsForSequelizeTables = {
+  /**
+   * If true, performs a hard deletion even when softDeletionEnabled is true.
+   * This permanently removes the record from the database.
+   *
+   * This is only available when softDeletionEnabled is true.
+   */
+  forceHardDeletion?: boolean;
```

Comment:
Let's tweak the docstring a bit to make it more scary when used
```suggestion
  /**
   * **THINK TWICE WHETHER YOU WANT THIS**
   * 
   * If `true`, performs a hard deletion even when `softDeletionEnabled` is `true`,
   * i.e. will hard delete records in tables which are otherwise soft deleted.
   *
   * This permanently removes the record from the database and only affects
   * tables with `softDeletionEnabled` set to `true`.
   */
  forceHardDeletion?: boolean;
```